src/widgets/powered-by/powered-by.js

```diff
@@ -62,7 +62,7 @@ export default function poweredBy({
   theme = 'light',
 } = {}) {
   if (!container) {
-    throw new Error(withUsage('The "container" option is required.'));
+    throw new Error(withUsage('The `container` option is required.'));
```

Comment:
at a later point these messages can be factorised out as well I think

Reply:
Yeah – we'll see how far we want to go!